OrbitCore/CoreApp.h

```diff
@@ -68,6 +69,10 @@ class CoreApp {
                        const MemoryCallback& callback) {
     process_memory_client_->GetRemoteMemory(pid, address, size, callback);
   }
+  // TODO(kuebler): hide this
```

Comment:
What exactly does this TODO mean? Should this be hidden now?

Also add an empty line above.

Reply:
For the symbol client, there is a LoadRemoteModules function in `App.h`, that calls the client. So the getter of the client can be protected. 
Here, I made the frame-pointer client accessible directly.

Reply:
Can you add a similar explanation to the TODO comment?

Reply:
Sure